Code/Framework/AzCore/AzCore/std/concepts/concepts.h

```diff
@@ -141,25 +144,28 @@ namespace AZStd
     /*concept*/ constexpr bool assignable_from = Internal::assignable_from_impl<LHS, RHS>;
 
     template<class T, class... Args>
-    /*concept*/ constexpr bool constructible_from = destructible<T> && is_constructible_v<T, Args...>;
```

Comment:
Not really , because we would have to adjust the entire statement itself.
Plus it is moot once O3DE is built with C++20, because we can alias the standard concepts into the AZStd namespace.